supporting_scripts/analysis-of-endpoint-connections/src/main/java/de/tum/cit/endpointanalysis/RestCallAnalyzer.java

```diff
@@ -0,0 +1,144 @@
+package de.tum.cit.endpointanalysis;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class RestCallAnalyzer {
```

Comment:
This seams like some code is duplicated here. Would it be worth introducing a common super class for this class and the EndpointAnalyzer?

Reply:
You are right, and I am already looking into that. As I have not yet come up with a satisfactory solution, I would like to move this into a follow-up.